contributor_docs/access.md

```diff
@@ -0,0 +1,34 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means we prioritize the needs of marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is not the first conversation we've had about access and inclusion. We share similar intentions in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
+
+**Please consider this a starting point.** We want to invite more conversations about what access means and how we can prioritize it.
+
+## Kinds of access
+
+Increasing access is not a focus on expanding the raw number of people in the p5.js community. It is a focus on making p5.js available to and approachable for more people who are excluded from the p5.js community (intentionally or not) and people who are excluded from similar tools.
+
+Access here means making p5.js better for:
+
+- People who speak Spanish, Chinese, or other languages besides English
+- People who are Black, Indigenous, or people of Color
+- People who are lesbian, gay, bisexual, trans, or queer
+- People with marginalized genders
+- People who are disabled or sick
```

Comment:
+ An item about income / class

Reply:
- also add item about rural communities

Reply:
For the first point above, maybe:
- People who lack opportunities and/or resources to engage with creative coding due to class or income

Reply:
@lm-n what about
People from the Global South
?

Reply:
i added the class/income language from @lmccart :)

Reply:
@montoyamoraga I feel uncomfortable with Global South... @dorothysantos do you have any ideas?

Reply:
@drsantos8791 just realized I tagged your other github account. Can you help us with the people that are not from North America/Europe?

Reply:
@lm-n Not a problem! It was easier for me to create a PF-specific GitHub account. Pardon the confusion. 

As for the "Global South," I understand your discomfort because in places such as the Philippines, for instance, the country (and in the education and tech spheres/realms) experience "brain drain" (not the best terminology, but this phenomenon has been documented). Essentially, even within the Filipino community, class is a huge issue that can't be easily remedied by support from a US-based program because you may be providing this opportunity to someone who readily has access to these opportunities within their region or country. This topic is such a thorny one, to be honest. I have too many thoughts for a comment, honestly. But I can see how we might flatten experiences and this is why transnational perspectives are super important. @montoyamoraga @lmccart @xinemata We can discuss for the future! Definitely an important issue especially as we think about translation projects.

Reply:
@lm-n @montoyamoraga @drsantos8791 this doesn't address all the issues here by any means, but could "non-western" be a useful term?

Reply:
@lmccart I definitely think "non-western" is better than "Global South"! But in translation it might get lost, in Spanish language the idea of the "west" includes Latin America and the Caribbean while in American English these regions are "non-western."